drivers/sensor/rpi_pico_temp/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Raspberry Pi Pico temperature sensor configuration options
+
+# Copyright (c) 2023 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config RPI_PICO_TEMP
+	bool "Raspberry Pi Pico CPU Temperature Sensor"
+	default y
+	depends on DT_HAS_RASPBERRYPI_PICO_TEMP_ENABLED
+	depends on (ADC && SOC_SERIES_RP2XXX)
```

Comment:
You already have a dependency on the dts compatible, so I think you can remove the redundant dependency on the SOC_SERIES

Reply:
It make a sense.
Removed SOC_SERIES_RP2XXX.